scripts/workers/submission_worker.py

```diff
@@ -361,7 +361,15 @@ def extract_submission_data(submission_id):
         input_file = submission.submission_input_file
     else:
         input_file = submission.input_file
-    submission_input_file = input_file.url
+    try:
+        submission_input_file = input_file.url
+    except Exception as e:
+        logger.exception(
+            "{} Submission input file does not exist for submission {}, error {}".format(
+                SUBMISSION_LOGS_PREFIX, submission_id, e
+            )
+        )
+        return None
```

Comment:
This case should never happen. `input_file` should never be `None`. This needs to be fixed when making a submission/updating it. We can;'t handle it in submission worker.

Reply:
Yeah agreed, this should not happen here. 
This is a rare thing and the events which I interpreted were:
- Both the requests Submission file from the side-car container and "submission.json/csv not found" from the code_upload worker happened in parallel.
[Status update request] fetched submission object which did not have submission_input_file.
[Submisison file request] came and updated the db and created a message and returned.
[Status update request] used the variable in memory that had submission_input_file as None.

Added the exception handling here just in case. I totally agree we should take care of this.

Reply:
If we have less time_delta than side-car container would exit as soon as the submission container and this situation would not occur. This is also one of the reasons of having less time_delta.